### ISSUE TITLE
Error in the example-app

### ISSUE THREAD

**Issue by bastianwegge** [simukappu/activity_notification#19]
Hey guys,

I like the gem very much and we'd love to use it in one of our projects. Currently while testing your example 'rails-app' in the specs folder, I ran into the following error:

![x](https://cloud.githubusercontent.com/assets/1541474/25747045/60e2a0ce-31a6-11e7-9751-675b20b22054.png)

I guess this is not on purpose but maybe I don't understand this good enough.

**Comment by simukappu** [comment]
Hi,

It seems that `admins` table in your local database is empty and ActivityNotification catches nil class as notification targets for `:admins`.  I think there is a little problem in test app configuration. You can work this test app with empty `admins` table if you update line 39 in `spec/rails_app/app/models/comment.rb`
```ruby
  acts_as_notifiable :admins, targets: [Admin.first],
```
to
```ruby
  acts_as_notifiable :admins, targets: [Admin.first].compact,
```

I will update test app in master branch.

Thanks!

**Comment by bastianwegge** [comment]
As I'm digging deeper into the project, I've found another thing concerning the subscriptions. Maybe this has to do with the postgres adapter?

![error2](https://cloud.githubusercontent.com/assets/1541474/25766174/18061412-31f1-11e7-8de5-63faec6197e2.png)

I have setup the `routes.rb` setup:
```ruby
notify_to :users, with_devise: :users, with_subscription: { controller: 'users/subscriptions' }
```

The model `user.rb`:
```ruby
  acts_as_target email: :email

  def unopened_notifications
    notifications.unopened_only
  end
```

And the notifiable model `my_object.rb`:
```ruby
  acts_as_notifiable :users,
    targets: ->(my_object) {
      [my_object.user]
    },
    notifiable_path: :my_object_notifiable_path

  def my_object_notifiable_path
    my_object_path(self)
  end
```

All of this is taken from the documentation. Maybe you can point me in the direction. I would also love to contribute, if you have any starting-point for me.

**Comment by simukappu** [comment]
This is really helpful report!
I have found `ActivityNotification`'s bug in ActiveRecord query for Postgres database. We have to update line 178 in `lib/activity_notification/orm/active_record/notification.rb` and line 46 in `lib/activity_notification/orm/active_record/subscription.rb`.
I have fixed this issue and updated master branch.

We are using this gem with MySQL and have been testing with SQLite and MongoDB. I have just updated test for MySQL and Postgres database and made them run at Travis CI. I would appreciated it if you could use this gem with Postgres and contribute us!

Please try to test with latest master branch. Thank you!

**Comment by bastianwegge** [comment]
Checked again, just to give you feedback: it works!

Thank you so much!

**Comment by simukappu** [comment]
I have just released the gem version 1.4.0 including these bug fixes.
Please try to use this if you want.

Thanks!
